src/ws/cockpitauth.c

```diff
@@ -761,7 +850,8 @@ cockpit_auth_remote_login_async (CockpitAuth *self,
                                     cockpit_auth_remote_login_async);
 
```

Comment:
It would be nice to generalize this local-ssh stuff and fit it into the same configuration scheme as well.